### PR TITLE
K8s KPIs with Kuberhealthy blog post

### PR DESCRIPTION
**Pull request by joshulyne**
Hello! I'm one of maintainers of the Kuberhealthy project -- an operator for synthetic monitoring on Kubernetes. I'm finally back with a rewrite for the Kuberhealthy blog post.  So sorry it took so long, and thank you again for that feedback on the previous post @mrbobbytables @onlydole! With this post we aimed to talk not just on how to install kuberhealthy to your k8s cluster, but also how to integrate it with prometheus so that you can start collecting cluster kpis.

### CODE REVIEW

File: content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+issues and contributed to discussions on the #kuberhealthy Slack channel. We quickly set to work to address all your feedback 
+with a newer version of Kuberhealthy. Additionally, we created a guide on how to easily install and use Kuberhealthy in order to capture some helpful synthetic [KPIs](https://kpi.org/KPI-Basics). 
+
+### Deploying Kuberhealthy
+
+To install Kuberhealthy, make sure you have [Helm 3](https://helm.sh/docs/intro/install/) installed. If not, you can use the generated flat spec files located 
+in this [deploy folder](../deploy). You should use [kuberhealthy-prometheus.yaml](../deploy/kuberhealthy-prometheus.yaml) if you don't use the [Prometheus Operator](https://github.com/coreos/prometheus-operator), and [kuberhealthy-prometheus-operator.yaml](../deploy/kuberhealthy-prometheus-operator.yaml) if you do.  If you don't use Prometheus at all, you can still use Kuberhealthy with a JSON status page and/or InfluxDB integration using [this spec](../deploy/kuberhealthy.yaml).
```
Comment (sftim) [inline content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md:21]: Links like `../deploy` aren't likely to work.

File: content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+When enabling Prometheus (not the operator), the Kuberhealthy service gets the following annotations added:
+```.env
+prometheus.io/path: /metrics
+prometheus.io/port: "80"
+prometheus.io/scrape: "true"
+```
+
+In your prometheus configuration, add the following example scrape_config that scrapes the Kuberhealthy service given the added prometheus annotation:
+
+```
```
Comment (sftim) [inline content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md:129]: You could mark this as YAML

File: content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+We define availability as the K8s cluster control plane being up and functioning as expected. This is measured by our ability to create a deployment, do a rolling update, and delete the deployment within a set period of time. 
+
+We calculate this by measuring Kuberhealthy's [deployment check](https://github.com/Comcast/kuberhealthy/tree/master/cmd/deployment-check) successes and failures. 
+  - Availability = Uptime / (Uptime * Downtime)
+  - Uptime = Number of Deployment Check Passes * Check Run Interval
+  - Downtime = Number of Deployment Check Fails * Check Run Interval
+  - Check Run Interval = how often the check runs (`runInterval` set in your KuberhealthyCheck Spec)
+
+- PromQL Query (Availability % over the past 30 days): 
+  ```
```
Comment (sftim) [inline content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md:177]: You could label this as promql

File: content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
+---
+layout: blog
+title: “K8s KPIs with Kuberhealthy”
```
Comment (sftim) [inline content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md:3]: This looks like the wrong quote characters.

File: content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+- [dns-status-internal](https://github.com/Comcast/kuberhealthy/tree/master/cmd/dns-resolution-check): Validates that internal cluster DNS is functioning as expected.
+
+To view other available external checks, check out the [external checks registry](https://github.com/Comcast/kuberhealthy/blob/master/docs/EXTERNAL_CHECKS_REGISTRY.md) where you can find other yaml files you can apply to your cluster to enable various checks.
+
+Kuberhealthy check pods should start running shortly after Kuberhealthy starts running (1-2 minutes). Additionally, the check-reaper cronjob runs every few minutes to ensure there are no more than 5 completed checker pods left lying around at a time.
+
+To get status page view of these checks, you'll need to either expose the `kuberhealthy` service externally by editing the service `kuberhealthy` and setting `Type: LoadBalancer` or use `kubectl port-forward service/kuberhealty 8080:80`. When viewed, the service endpoint will display a JSON status page that looks like this:
```
Comment (sftim) [inline content/en/blog/_posts/K8s-KPIs-with-Kuberhealthy.md:67]: nit: `kubectl port-forward service/kuberhealty 8080:80` has a typo in the service name.

**Comment by joshulyne** [comment]
@sftim thanks for the thorough review, hopefully i addressed everything! 
@mrbobbytables appreciate the fast review!

**Comment by castrojo** [comment]
Ok just need two more things: Can you update the date to tomorrow or sometime later this week? Otherwise it'll publish it in the past before the current article on the blog and it won't show up on the front page which would be unfortunate. 

You'll also need to add the date to the filename as well,  so like `2020-05-28-K8s-KPIs-with-Kuberhealthy.md`

**Comment by mrbobbytables** [comment]
/approve
/hold
can release tomorrow for publishing 👍

File: content/en/blog/_posts/2020-05-29-K8s-KPIs-with-Kuberhealthy.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+  ```
+##### 2. Set your current namespace to "kuberhealthy": 
+  ```
+  kubectl config set-context --current --namespace=kuberhealthy 
+  ```
+##### 3. Add the kuberhealthy repo to Helm: 
+  ```
+  helm repo add kuberhealthy https://comcast.github.io/kuberhealthy/helm-repos
+  ```
+##### 4. Dependng on your Prometheus implementation, install Kuberhealthy using the appropriate command for your cluster:
```
Comment (onlydole) [inline content/en/blog/_posts/2020-05-29-K8s-KPIs-with-Kuberhealthy.md:36]:
```suggestion
##### 4. Depending on your Prometheus implementation, install Kuberhealthy using the appropriate command for your cluster:
```

**Pull request merged**